tests/performance/reindexing/reindexing_and_feeding.rb

```diff
@@ -0,0 +1,255 @@
+
+# coding: utf-8
+# Copyright 2020 Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+require 'performance_test'
+require 'app_generator/search_app'
+
+class ReindexingAndFeeding < PerformanceTest
+
+  def initialize(*args)
+    super(*args)
+  end
+
+  def timeout_seconds
+    900
+  end
+
+  def setup
+	  super
```

Comment:
Indentation is off. Should be 2 spaces. Applies for lines further down as well.

Reply:
Argh. Will fix.